internal/repository/repository.go

```diff
@@ -16,10 +16,10 @@ const (
 )
 
 type Service struct {
```

Comment:
if we don't want to do the refactor repository/state now, maybe we should put in a TODO

Reply:
ok